textual_fspicker/parts/drive_navigation.py

```diff
@@ -0,0 +1,73 @@
+"""Provides a widget for drive navigation."""
+
+##############################################################################
+# Python imports.
+import os
+import pathlib
+from dataclasses import dataclass
+
+##############################################################################
+# Textual imports.
+from textual import on
+from textual.app import App, ComposeResult
+from textual.message import Message
+from textual.widgets import OptionList, Static
+
+
+##############################################################################
+class DriveNavigation(OptionList):
+    DEFAULT_CSS = """
+    DriveNavigation, DriveNavigation:focus {
+        border: blank;
+        border-right: $panel-darken-1;
+        width: 10;
+        height: 100%;
+    }
+    """
+    """Default styling for the widget."""
+
+    @dataclass
+    class DriveSelected(Message):
+        """Message sent when a drive is selected."""
+
+        drive_root: str
+
+    def on_mount(self) -> None:
+        """Add available drives to the widget."""
+        self.add_options(os.listdrives())
+
+    @on(OptionList.OptionSelected)
+    def select_drive(self, event: OptionList.OptionSelected) -> None:
+        """Post a DriveSelected message.
+
+        Args:
+            event (OptionList.OptionSelected): the drive selected event from the parent
```

Comment:
```suggestion
            event: the drive selected event from the parent
```

It's best to not include type information in the docstring.